src/org/usfirst/frc4904/robot/commands/RollersIntake.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.CrateIO;
+import edu.wpi.first.wpilibj.command.Command;
+
+
+public class RollersIntake extends Command {
+	
+	protected void execute() {
+		set(CrateIO.INTAKE_SPEED);
+	}
+
+	private void set(double intakeSpeed) {
+		
+		RobotMap.Component.CubeIOroller.set(intakeSpeed);
```

Comment:
Take a look at [MotorSet.java](https://github.com/RoboticsTeam4904/standard/blob/master/commands/motor/MotorSet.java)
I think you should keep the speed stored here locally, like it is in MotorSet.java, rather than on `CubeIORoller`.
The problem is that when you set the speed on the subsystem from this command, that speed persists outside the context of this command. If this command sets the speed to 1 and ends, the speed will go to 0 with the default command. If you made another instance of this command (`new RollersIntake();`) then it would resume the speed of 1.

Again, look at  [MotorSet.java](https://github.com/RoboticsTeam4904/standard/blob/master/commands/motor/MotorSet.java) to see how the speed is stored locally there.